<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PistonClass</class>
 <widget class="QMainWindow" name="PistonClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3D GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="margin">
     <number>1</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <widget class="QLabel" name="label_2">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>111</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>Length of piston (mm)</string>
             </property>
            </widget>
            <widget class="QSplitter" name="splitter">
             <property name="geometry">
              <rect>
               <x>140</x>
               <y>10</y>
               <width>91</width>
               <height>20</height>
              </rect>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QDoubleSpinBox" name="_dsbLength">
              <property name="minimum">
               <double>1.000000000000000</double>
              </property>
              <property name="maximum">
               <double>999.990000000000009</double>
              </property>
              <property name="value">
               <double>150.000000000000000</double>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <widget class="QLabel" name="label_3">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>101</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>Diameter of piston</string>
             </property>
            </widget>
            <widget class="QSplitter" name="splitter_3">
             <property name="geometry">
              <rect>
               <x>140</x>
               <y>10</y>
               <width>91</width>
               <height>20</height>
              </rect>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QDoubleSpinBox" name="_dsbDiameter">
              <property name="minimum">
               <double>1.000000000000000</double>
              </property>
              <property name="maximum">
               <double>999.990000000000009</double>
              </property>
              <property name="value">
               <double>190.000000000000000</double>
              </property>
             </widget>
            </widget>
            <zorder>splitter_3</zorder>
            <zorder>label_3</zorder>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <widget class="QLabel" name="label">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>81</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>Thickness (mm)</string>
             </property>
            </widget>
            <widget class="QSplitter" name="splitter_4">
             <property name="geometry">
              <rect>
               <x>140</x>
               <y>10</y>
               <width>91</width>
               <height>20</height>
              </rect>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QDoubleSpinBox" name="_dsbThickness">
              <property name="minimum">
               <double>0.000000000000000</double>
              </property>
              <property name="maximum">
               <double>999.990000000000009</double>
              </property>
              <property name="value">
               <double>12.000000000000000</double>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_5" native="true">
            <widget class="QLabel" name="label_4">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>81</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>Number of rings</string>
             </property>
            </widget>
            <widget class="QSplitter" name="splitter_5">
             <property name="geometry">
              <rect>
               <x>140</x>
               <y>10</y>
               <width>91</width>
               <height>20</height>
              </rect>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QSpinBox" name="_spbNumberRing">
              <property name="value">
               <number>3</number>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <widget class="QPushButton" name="_bntMake">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>10</y>
               <width>75</width>
               <height>23</height>
              </rect>
             </property>
             <property name="text">
              <string>Make</string>
             </property>
            </widget>
            <widget class="QPushButton" name="_bntExport">
             <property name="geometry">
              <rect>
               <x>150</x>
               <y>10</y>
               <width>75</width>
               <height>23</height>
              </rect>
             </property>
             <property name="text">
              <string>Export</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="_referenceImage">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="text">
         <string>3D Viewer</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="_glWidget">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="piston.qrc"/>
 </resources>
 <connections/>
</ui>
